Translate descriptions

DahliaSR opened the pull request:
Closes #1

DahliaSR committed:
Prepared Translation

- added folder structure for german translatin
- copied original files

DahliaSR committed:
translated short description

part of #1

DahliaSR committed:
translated description

Closes #1

GiForce reviewed (approved):
🌟

GiForce commented on translations/de/task_short_description.html:
```diff
@@ -1 +1,2 @@
-Division, great for numbers, food, labor, and buried treasure. Not so much for politics.
\ No newline at end of file
+Teilen, großartig bei Zahlen, Essen, Arbeit und vergrabenen Schätzen. Nicht so sehr in der
```
Mich stört: _großartig bei Zahlen, ..._ aber: [großartig] _nicht so sehr in der Politik_.
"Für" würde besser klingen, geht für beides. Für die Politik ist das Teilen nicht grossartig. Für Zahlen schon, auch für vergrabene Schätze. Obwohl mit den Zahlen habe ich gerade etwas Mühe, es ist ja für den Anwender großartig. Mal sehen, Teilen ist großartig für Zahlen, um Sie in kleinere und handlichere Teile zu verwandeln. Zahlen waren noch nie so lieb wie nach dem Teilen ^^ Für sich allein wäre "mit" hier am besten. Für was soll Teilen gut sein? Für Zahlen, Essen, Arbeit. Für die Arbeit großartig, schneller und effizienter erledigt. Und hätten Schätze Gefühle, fänden Sie es auch grossartig mehr unter die Leute zu kommen ^^ Mhh, vielleicht geht ja "mit" als Universallösung? Ich überlasse Dir die Entscheidung.

DahliaSR replied:
Da weder Zahlen, Essen, Arbeit noch Schätze etwas davon haben geteilt zu werden funktioniert **_für_** hier nicht wirklich.

**_Mit_** geht auch nicht nicht wirklich... Denn das Problem liegt ja nicht bei der Politik, sondern bei den Politikern.

Wie wärs hiermit:  
_Teilen ist großartig bei Zahlen, Essen, Arbeit und vergrabenen Schätzen. In der Politik ist es aber unbeliebt._

GiForce reviewed (changes requested):
.

GiForce commented:
Naja, ich dachte mit "für" meint man in diesem Fall eben nicht das diese Dinge an sich etwas davon haben, sondern die Auseinandersetzung mit diesen Dingen, zum Beispiel kann man ja auch sagen "Jene Jacke ist großartig für lange Regenspaziergänge".
Ob das Problem bei der Politik oder den Politikern liegt, ist das nicht beinahe eine philosophische Frage?
Aber Dein Vorschlag birgt eine großartige Lösung; "beliebt" anstelle großartig schon im ersten Teil, etwa so
Teilen ist beliebt bei Zahlen, Essen, Arbeit und vergrabenen Schätzen. In der Politik eher weniger. 
Ich glaube so ist das jetzt schön elegant gesagt.
🥂

> On 6 Jun 2017, at 19:14, DahliaSR <notifications@github.com> wrote:
> 
> @DahliaSR commented on this pull request.
> 
> In translations/de/task_short_description.html:
> 
> > \ No newline at end of file
> +Teilen, großartig bei Zahlen, Essen, Arbeit und vergrabenen Schätzen. Nicht so sehr in der
> Da weder Zahlen, Essen, Arbeit noch Schätze etwas davon haben geteilt zu werden funktioniert für hier nicht wirklich.
> 
> Mit geht auch nicht nicht wirklich... Denn das Problem liegt ja nicht bei der Politik, sondern bei den Politikern.
> 
> Wie wärs hiermit:
> Teilen ist großartig bei Zahlen, Essen, Arbeit und vergrabenen Schätzen. In der Politik ist es aber unbeliebt.
> 
> —
> You are receiving this because your review was requested.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

DahliaSR committed:
Correcttions

According to [Review](https://github.com/CheckIO-to-German/The-Greatest-Common-Divisor/pull/2#pullrequestreview-41950843)

DahliaSR commented:
@GiForce 
Approval, pls!

GiForce reviewed (approved):
💫
